src/DistributedLock.Tests/Infrastructure/Redis/RedisServer.cs

```diff
@@ -35,7 +35,12 @@ private RedisServer(int? port, bool allowAdmin)
                 .RedirectStandardErrorTo(Console.Error);
             ActiveServersByPort.Add(this.Port, this);
         }
-        this.Multiplexer = ConnectionMultiplexer.Connect($"localhost:{this.Port}{(allowAdmin ? ",allowAdmin=true" : string.Empty)}");
+        this.Multiplexer = ConnectionMultiplexer.Connect(new ConfigurationOptions
```

Comment:
https://github.com/StackExchange/StackExchange.Redis/blob/5504ed9a93f729f204e6cb53280cb4efa1ccd390/docs/Configuration.md says that there are two ways to set configuration. I changed this to using `ConfigurationOptions` because `BacklogPolicy` is one of the code-only options.